vernac/comAssumption.ml

```diff
@@ -220,8 +220,16 @@ let do_assumptions ~program_mode ~poly ~scope ~kind ?user_warns ~inline l =
   let univs = Evd.check_univ_decl ~poly sigma udecl in
   declare_context ~try_global_assum_as_instance:false ~scope ~univs ?user_warns ~inline ctx
 
+let warn_context_outside_section =
+  CWarnings.create ~name:"context-outside-section"
+    ~category:Deprecation.Version.v8_20
+    (fun () -> Pp.strbrk "Use of \"Context\" outsdide sections is \
```

Comment:
```suggestion
    (fun () -> Pp.strbrk "Use of \"Context\" outside sections is \
```

This being said, the question of removing `Context` outside of a section is more disputable as the replacement will be a priori more verbose (including possibly requiring an `Existing`). Maybe can we find another word.

Reply:
Well, we'd ultimately like to see Variable/Hypothesis and Context converge, so we'd rather not introduce yet another subtle difference between the two.